index.js

```diff
@@ -1269,6 +1269,7 @@ class Encore {
      *              Configure the loader to lint `.vue` files
      * ```
      *
+     * @deprecated Prefer using "Encore.enableEslintPlugin()" instead.
```

Comment:
Should we actually trigger a deprecation log?

Reply:
Yeah why not!